integration_tests/test_asset_group_sightings.py

```diff
@@ -356,11 +250,27 @@ def test_asset_group_sightings(session, login, codex_url, test_root):
     sighting_guid = response.json()['guid']
     assert response.json() == {
         'guid': sighting_guid,
+        'id': sighting_guid,
```

Comment:
Do we still need both 'guid' and 'id'?

Reply:
See above, right now yes, Real soon now, no